specification/loadtestservice/data-plane/Microsoft.LoadTestService/readme.md

```diff
@@ -0,0 +1,117 @@
+# loadtestservice
```

Comment:
could we rename this folder into specification/loadtestservice/data-plane/loadtesting/ I don't think we have the Microsoft.LoadTestService in data plane apis?

Reply:
Had kept this to avoid any breaking change. I'll do this if it's a no-risk operation.